interface/client/lib/helpers/helperFunctions.js

```diff
@@ -50,6 +50,21 @@ Helpers.formatUrl = function(url){
 };
 
 /**
+Sanatizes URLs to prevent phishing and XSS attacks
+
+@method sanatizeUrl
+@param {String} url
+**/
+Helpers.sanatizeUrl = function(url){    
+    // add http:// if no protocol is present
+    if(_.isString(url)) {
+        url = url.replace(/^(javascript\:)?(data\:)?\/\//i,'http:');
```

Comment:
The regex doesn't seem to be valid. It defines two capture groups but won't make use of them. Thus the `replace` will omit the `//` and output: `http:victim.com/superlegit/...`.

It currently looks like this:

<img width="457" alt="bildschirmfoto 2016-11-17 um 23 58 41" src="https://cloud.githubusercontent.com/assets/10135872/20411455/d7789e82-ad21-11e6-96b6-68d7e8dfef3c.png">

Instead we might use this regex:

<img width="350" alt="bildschirmfoto 2016-11-17 um 23 58 54" src="https://cloud.githubusercontent.com/assets/10135872/20411466/e5dbc094-ad21-11e6-9384-bdbe5aaccc6d.png">

`url = url.replace(/^(?:javascript|data)(?=:\/\/)/i, 'http');`

Note:
 - `(?:group)` will make it non-capture as it is not needed
 - `foo(?=bar)` is a positive lookahead and will only pass "foo" to the replace function